pkg/config/api-server/config.go

```diff
@@ -56,19 +58,29 @@ type ApiServerHTTPSConfig struct {
 	TlsCertFile string `yaml:"tlsCertFile" envconfig:"kuma_api_server_https_tls_cert_file"`
 	// Path to TLS key file. Autoconfigured from KUMA_GENERAL_TLS_KEY_FILE if empty
 	TlsKeyFile string `yaml:"tlsKeyFile" envconfig:"kuma_api_server_https_tls_key_file"`
+	// TlsMinVersion defines the minimum TLS version to be used
+	TlsMinVersion string `yaml:"tlsMinVersion" envconfig:"kuma_api_server_https_tls_min_version"`
```

Comment:
Do we want `TlsMaxVersion` ? AFAIK the negotiation will select the highest version available and at Allegro we've seen lower performance of 1.3 (on that hardware) in regards to 1.2 so we [specifically opted for 1.2](https://github.com/allegro/envoy-control/blob/912799d6f7961f7dce5fb6279cd45fba692d54b0/envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt#L122-L123) (I know that this is for the DP but I guess the same rules apply).

Reply:
I thought about it and then thought if you support it why would you for downgrade. I guess perf is a valid argument, I'll add it